opengrok-indexer/src/test/java/org/opengrok/indexer/history/PerforceRepositoryTest.java

```diff
@@ -40,36 +41,38 @@
 
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertEquals;
-import org.opengrok.indexer.configuration.RuntimeEnvironment;
 import static org.opengrok.indexer.history.PerforceRepository.protectPerforceFilename;
+import static org.opengrok.indexer.history.PerforceRepository.unprotectPerforceFilename;
 
 /**
  * Do basic testing of the Perforce support
  *
  * @author Trond Norbye
  */
-@ConditionalRun(RepositoryInstalled.PerforceInstalled.class)
 public class PerforceRepositoryTest {
 
     @Rule
     public ConditionalRunRule rule = new ConditionalRunRule();
     
     private static boolean skip;
     private static List<File> files;
-    private static final File root = new File("/export/opengrok_p4_test");
+    private static final File root = new File("/var/opengrok/src/p4foo");
```

Comment:
I assume this needs some setup steps to populate the root somehow. Could you perhaps update the test case with the steps ?

Reply:
OK done